Translations/bg.po

```diff
@@ -367,8 +367,8 @@ msgstr "Отказ"
 msgid "Client-Server (TCP)"
 msgstr "Клиент-сървър (TCP)"
 
-#: Source/DiabloUI/selconn.cpp:15
-msgid "Loopback"
+#: Source/DiabloUI/selconn.cpp:15 # Needs updated translation
```

Comment:
Translations should be marked as fussy, this looks like a manually added comment?

Reply:
Not sure what you mean. Yeah I added those comments manually

Reply:
https://til.hashrocket.com/posts/wcmc04nort-fuzzy-translation-key-merging-with-gettext